src/app/components/routing/application-routing.component.ts

```diff
@@ -22,6 +22,9 @@ export class ApplicationRoutingComponent implements OnInit{
     if(userRole.indexOf('jps-recorder') != -1){
       void this.router.navigate(['sittingRecords', 'manage'])
     }
+    else {
```

Comment:
In this block, we need to make sure that the user cant navigate to manage straight away if the user isn't a recorder role.

Reply:
Done. Added the check